src/Collection/FilesCollection.php

```diff
@@ -104,6 +104,24 @@ public function notPath($pattern)
     }
 
     /**
+     * Adds rules that filenames must not match.
+     *
+     * You can use patterns (delimited with / sign) or simple strings.
+     *
+     * $collection->notPaths(['/^spec\/','/^src\/'])
+     *
+     * @param array $pattern
+     *
+     * @return FilesCollection
+     */
+    public function notPaths(array $pattern)
```

Comment:
Can you change `notPath()` to use this method (similar to `path()` and `paths()`) ?